src/Repository.js

```diff
@@ -61,6 +61,14 @@ class Repository {
 
   get packageConfigs() {
     if (this.lernaJson.useWorkspaces) {
+      if (!this.packageJson.workspaces) {
+        throw new ValidationError(
+          "Invalid workspaces",
+          "workspaces need to be defined in the root package.json." +
```

Comment:
This should be a multiline tagged template string (using `dedent`, several examples of usage in src/Command.js).

Should probably call them "yarn workspaces", too, to clarify their origin.